posydon/grids/post_processing.py

```diff
@@ -70,13 +71,13 @@ def print_CC_quantities(EXTRA_COLUMNS, star, MODEL_NAME=None):
             print(format_val.format(MODEL_NAME,
                     star.state, star.SN_type, star.f_fb,
                     star.mass, spin, star.m_disk_accreted,
-                    star.m_disk_radiated))
+                    star.m_disk_radiated, star.M4, star.mu4))
```

Comment:
We'd need to add a check, whether those values are None. Cf. the check for the spin, otherwise we'll get a lot of exceptions for this try.

It would be even better to restructure the print out function here completely -- but I think that's beyond the scope of this PR.

Reply:
ok yes. Why dont we do the same for m_disk_accreted, m_disk_radiated?

Reply:
They are 0 not None, but we could do for them, too.